pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -7101,7 +7225,7 @@ def prepare_fitting_slice(self, freqs, qbn, mu_guess,
         offset_guess = data_slice[0]
         GaussianModel.set_param_hint('sigma',
                                      value=self.sigma_guess[qbn],
-                                     vary=False)
+                                     vary=True)
```

Comment:
should we give the user the option to decide whether sigma should be fitted or fixed (to a user-provided value)? Maybe fitting it is less robust in some instances.

Reply:
I've fixed this by implementing a general way for the user to update guess pars. See the new method `set_user_guess_pars` in `BaseDataAnalysis`. Let me know what you think!

Reply:
I think the concept makes sense. I haven't done tests now, but if you say it works as intended, I am fine with it. Thank you!